data/weapons/weapon_data.csv

```diff
@@ -260,10 +260,10 @@ Micro Swarmer,diableavionics_itanolauncher,,,,750,,200,,5,6,5,4,0.2,4,FRAGMENTAT
 ,diableavionics_valiant_cover_RIGHT,,,,1,1,,,,,,,,,,,,,10,,,,,,,,,,,,,,SYSTEM,,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 #ZEPHYR,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Burchel (long delay),diableavionics_zephyr_head,,,1500,400,200,,,,36,9,6,0.75,2,ENERGY,,,0,0,0.48,0.02,,,,,1000000,,,,,,,"PD,PD_ONLY,SYSTEM",,,,,,,,,,,,,,,
-Micro Glaux,diableavionics_zephyr_leftBeam,2,0.5,3000,650,300,,,,75,8,3,1,,ENERGY,,,0,0,0.48,0.02,,,,,1000000,,,,,,,SYSTEM,"energy8,beam8",,,,,,,,,,,,,,
-Micro Glaux,diableavionics_zephyr_rightBeam,2,0.5,3000,650,300,,,,75,8,3,1,,ENERGY,,,0,0,0.48,0.02,,,,,1000000,,,,,,,SYSTEM,"energy8,beam8",,,,,,,,,,,,,,
-,diableavionics_zephyr_chestBeam,3,1,5000,850,1000,200,25,25,0,15,,,,ENERGY,,,3,0.25,2,10,,,,,1000000,,,,,,,SYSTEM,"energy20,beam20",,,,,,,,,,,,,,
+Burchel ,diableavionics_zephyr_head,,,1500,600,,150,,,36,9,12,1,1,ENERGY,,,0,0,0.48,0.02,,,,,1000000,,,,,,,"PD,ANTI_FTR,PD_ALSO,SYSTEM",,,,,,,,,,,,,,,
+Micro Glaux,diableavionics_zephyr_leftBeam,2,0.5,3000,650,300,,,,75,8,3,1,,ENERGY,,,0,0,0.48,0.02,,,,,1000000,,,,,,,"SYSTEM,PD,ANTI_FTR,PD_ALSO","energy8,beam8",,,,,,,,,,,,,,
+Micro Glaux,diableavionics_zephyr_rightBeam,2,0.5,3000,650,300,,,,75,8,3,1,,ENERGY,,,0,0,0.48,0.02,,,,,1000000,,,,,,,"SYSTEM,PD,ANTI_FTR,PD_ALSO","energy8,beam8",,,,,,,,,,,,,,
+,diableavionics_zephyr_chestBeam,3,1,5000,850,1500,300,25,25,0,15,,,,ENERGY,,,3,0.25,2,10,,,,,1000000,,,,,,,SYSTEM,"energy20,beam20",,,,,,,,,,,,,,
```

Comment:
Zephyr burchel set to 600 range (from 400), damage set to 150 (from 200), ammo max set to 12 (from 6), ammo regen set to 1 per second (from 0.75 per second), ammo reload size set to 1 (from 2)
Zephyr chestbeam set to 1500 dps (from 1000), damage per shot (not sure this matters) set to 300 (from 200)